AzureSentinel - keywords argument from 'comma separate' to 'str' in azure-sentinel-threat-indicator-query (#27380)
demisto/content: Packs/AzureSentinel/ReleaseNotes/1_5_8.md

#### Integrations

##### Microsoft Sentinel

> - Fixed an issue where ***azure-sentinel-threat-indicator-query*** command failed due to incorrect *keywords* argument..

Reviewer: ```suggestion
- Fixed an issue where ***azure-sentinel-threat-indicator-query*** would fail when provided with multiple keywords.
```
